www/activities/diary/js/diary.js

```diff
@@ -170,6 +170,33 @@ app.Diary = {
     el.innerText = dateString;
   },
 
+  sendStatistics: async function () {
+    if (app.Settings.get("statistics", "send-data") != true) {
+      return;
+    }
+
+    let entry = await this.getEntryFromDB(); // Get diary entry from DB
+
+    // Populate groups and get overal nutrition
```

Comment:
This comment here is not a good description of what the code does. Consider changing or removing it.

Reply:
true